internal/config/cgmgr/cgmgr.go

```diff
@@ -54,10 +56,17 @@ type CgroupManager interface {
 	// returns the cgroup path on disk for that containerID. If parentCgroup is empty, it
 	// uses the default parent for that particular manager
 	ContainerCgroupAbsolutePath(string, string) (string, error)
+	// PopulateContainerCgroupStats takes arguments sandbox parent cgroup, container ID, and
+	// containers stats object. It fills the object with information from the cgroup found
+	// given that parent and ID
+	PopulateContainerCgroupStats(string, string, *types.ContainerStats) error
```

Comment:
You can make it more self-documenting here by adding parameter names, e.g.
```go
PopulateContainerCgroupStats(sbParent, cid string, stats *types.ContainerStats) error
```

Reply:
good idea, done